packages/radio/src/mwc-radio-base.ts

```diff
@@ -147,6 +149,7 @@ export class RadioBase extends FormElement {
           <div class="mdc-radio__outer-circle"></div>
           <div class="mdc-radio__inner-circle"></div>
         </div>
+        <div class="mdc-radio__ripple" .ripple=${ripple()}></div>
```

Comment:
I guess we should do the same for `<mwc-button>`, where I created the new ripple node but didn't move the directive. It seemed to be working anyway, not sure I understand this directive though.

Reply:
I think the styling for the ripple is dependent only on the class names that the directive will create, but we probably should be mirroring the MDC template layout for maximum compatibility.

Reply:
I just tried moving the `<mwc-button>` `ripple` directive from the button to the ripple node, and the ripple stopped working. Instead, the whole button uniformly changes color on click, but there is no ripple effect.

Reply:
this ripple thing is hella confusing let me tell you

Reply:
Looking at their demo's code. I am wrong. Though it works, the ripple directive needs to stay on the root

Reply:
Can confirm, the ripple still works